Gemfile

```diff
@@ -47,8 +47,10 @@ group :development do
   # gem "spring"
 end
 
-# for authentication
 gem "devise", "~> 4.9"
 
-# for authorization
 gem "pundit", "~> 2.3"
+
```

Comment:
lines not required, fyi. There are couple conventions that you can handle with rubocop, can keep it this way for now. Let's start using rubocop from the next branches.